pom.xml

```diff
@@ -30,28 +30,28 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.642.1</jenkins.version>
```

Comment:
Maybe useful: the versions combination in this [pom file](https://github.com/jenkinsci/http-request-plugin/blob/master/pom.xml) worked for the http-request-plugin not too long ago.